prep-html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton Exercise</title>
+    <body>
+      <h1>I built a proper HTML skeleton!</h1>
+    </body>
+  </head>
```

Comment:
you want to make sure that the head only has the `title` element inside of it. 

The opening and closing body tags should be outside of the head tags, not within them.